vaadin-menu-bar-flow-parent/vaadin-menu-bar-testbench/src/main/java/com/vaadin/flow/component/menubar/testbench/MenuBarElement.java

```diff
@@ -64,4 +70,55 @@ private boolean isVisible(TestBenchElement element) {
                 "return arguments[0].style.visibility !== 'hidden'", element);
     }
 
+    /**
+     * Get TestBenchElements representing sub menu items under the first sub
+     * menu.
+     *
+     * @return List of TestBenchElements representing sub menu items.
+     */
+    public List<TestBenchElement> getSubMenuItems() {
+        return getSubMenuItems(getSubMenu());
+    }
+
+    /**
+     * Get TestBenchElements representing sub menu items under specific sub
+     * menu.
+     *
+     * @param overlay
+     *            The sub menu overlay from which items are being collected.
+     * @return List of TestBenchElements representing sub menu items.
+     */
+    public List<TestBenchElement> getSubMenuItems(TestBenchElement overlay) {
+        return overlay.$("vaadin-context-menu-item").all();
+    }
+
+    /**
+     * Get the sub menu overlay element.
+     *
+     * @return TestBenchElement for the first open sub menu.
+     */
+    public TestBenchElement getSubMenu() {
```

Comment:
I wonder if the name should be changed to e.g. `getSubMenuOverlay()` to make it clear?

There is `getSubMenu()` public API in the `MenuBar` that is used when creating items.
Some users might get confused by methods that are called the same, but with different meaning.

Reply:
I think it's fine as is, from a user perspective this element is effectively the "sub-menu", it being an overlay is more of a technical detail.